src/routes/+layout.svelte

```diff
@@ -16,12 +16,42 @@
     import { loading, requestedMigration } from './store';
     import { parseIfString } from '$lib/helpers/object';
     import Consent from '$lib/components/consent.svelte';
+    import { sdk } from '$lib/stores/sdk';
 
     if (browser) {
         window.VERCEL_ANALYTICS_ID = import.meta.env.VERCEL_ANALYTICS_ID?.toString() ?? false;
     }
 
     onMount(async () => {
+        // handle sources
+        if (browser && isCloud) {
```

Comment:
we dont need to check for browser, onmount only runs on browser anyway 👍🏻 

```suggestion
        if (isCloud) {
```